lib/uktimeconv/include/uk/timeutil.h

```diff
@@ -0,0 +1,64 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef __UK_TIMEUTIL_H__
+#define __UK_TIMEUTIL_H__ 1
```

Comment:
We usually have the following header guards layout (no defintion to `1` and closing `#endif` repeats the guard name with a comment):
```c
#ifndef __UK_TIMEUTIL_H__
#define __UK_TIMEUTIL_H__
```
...
```c
#endif /* __UK_TIMEUTIL_H__ */
```

Reply:
👌